flink-elastic-catalog/src/test/java/com/getindata/flink/connector/jdbc/catalog/ElasticCatalogTest.java

```diff
@@ -63,23 +63,23 @@ public static void beforeAll() throws Exception {
         createTestIndex(INPUT_PARTIAL_SCHEMA_TABLE_2, PARTIAL_SCHEMA_PATH_2);
         createTestIndex(INPUT_EMPTY_TABLE, INDEX_PATH);
         createTestIndex(INPUT_SPECIAL_CHARACTER_COLUMN_NAMES_TABLE, SPECIAL_CHARACTER_COLUMN_NAMES_INDEX_PATH);
-        TimeUnit.SECONDS.sleep(1);
+        TimeUnit.SECONDS.sleep(2);
```

Comment:
I think that one sleep after the last call of `addTestData` is enough. Could you test that?